packages/yoshi/src/verify-dependencies.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const chalk = require('chalk');
+const semver = require('semver');
+const importCwd = require('import-cwd');
+
+const { version } = require('../package.json');
```

Comment:
Ease readability later on

```suggestion
const { version: yoshiVersion } = require('../package.json');
```